src/iosrv/drpc_client.c

```diff
@@ -38,18 +38,18 @@ struct drpc *dss_drpc_ctx;
 static int
 notify_ready(void)
 {
-	Drpc__Call	req = DRPC__CALL__INIT;
+	Drpc__Call     *req;
 	Drpc__Response *resp;
 	int		rc;
 
-	req.module = DRPC_MODULE_SRV;
-	req.method = DRPC_METHOD_SRV_NOTIFY_READY;
+	req = drpc_call_create(dss_drpc_ctx,
+			DRPC_MODULE_SRV,
+			DRPC_METHOD_SRV_NOTIFY_READY);
```

Comment:
Do we need to check if req == NULL?

Reply:
Sorry, but would you mind aligning to "(" instead? (Since that's DAOS's coding style, I found a simple option that let Vim does the alignment automatically. Visual Studio Code should have something eq. I assume?)

Reply:
Good catch. Will fix both.